lib/chef/knife/cloud/google_service_helpers.rb

```diff
@@ -27,8 +27,8 @@ def create_service_instance
         zone:          locate_config_value(:gce_zone),
         wait_time:     locate_config_value(:request_timeout),
         refresh_rate:  locate_config_value(:request_refresh_rate),
-        max_pages:     locate_config_value(:max_pages),
-        max_page_size: locate_config_value(:max_page_size)
+        max_pages:     locate_config_value(:gce_max_pages),
```

Comment:
Not sure why we are renaming this but have a doubt that it may break if someone is using max_pages as value.

Reply:
@Vasu1105 we have already renamed this option `max_pages` to `gce_max_pages`, so I have fix this bug.